### ISSUE TITLE
No OSD for volume slider

### ISSUE THREAD

**Issue by D0J0P** [libretro/RetroArch#16234]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

When using RetroArch, whether you're in the menu or in any game, the volume slider OSD for Windows 10 doesn't show up when you lower or raise the volume.

### Expected behavior

That the volume slider would display in RetroArch when you lower and raise the volume

### Actual behavior

The volume slider doesn't display at all in Windows 10 when you lower and raise the volume. I have to Alt + Tab to another program window to see the exact volume level in Windows 10 and then go back to my game. 

### Steps to reproduce the bug

1. Open RetroArch
2. Press volume up and down on your keyboard
3. Notice that the volume slider doesn't show up, as if RetroArch is overriding it somehow. Alt + Tab to another program and then adjust the volume with the keyboard and see that it works properly outside of RetroArch. 

### Bisect Results

This was always an issue for me. I've started using RetroArch last year since version 1.15 I think.

### Version/Commit
1.17

- RetroArch: [version/commit]

### Environment information

- OS: Windows 10 Home
- Compiler: N/A

**Comment by Ryunam** [comment]
That might be a normal occurrence if you are using Exclusive Fullscreen (Settings -> Video -> Fullscreen Mode -> "Start in Fullscreen Mode" set to ON and "Windowed Fullscreen Mode" set to OFF).

It's expected to work that way, because by using Exclusive Fullscreen the application is taking full control of on-screen drawing and this bypasses the Windows OSD notifications as well. This usually allows the application to retain better framepacing and have less input latency.

**Comment by D0J0P** [comment]
You're right, it does bypass Windows OSD notifications in fullscreen. Do you think there will be a setting in the future to add OSD notifications in fullscreen?

**Comment by sonninnos** [comment]
This might be an interesting read regarding "exclusive" fullscreen and modern Windows:
https://devblogs.microsoft.com/directx/demystifying-full-screen-optimizations/

Especially this bit:
> With the release of Windows 10, we added Fullscreen Optimizations – which takes full screen exclusive games and runs them instead in a highly optimized borderless windowed format that takes up the entire screen. You get the visual experience and performance of running your game in FSE, but with the benefits of running in a windowed mode. These benefits include faster PC commands such as alt-tab, multiple monitor set ups and overlays. We have extensive performance data that indicates that almost all users who use Fullscreen Optimizations have equal performance to Full Screen Exclusive.
